src/lib/components/wizard-completion-step.component.ts

```diff
@@ -41,8 +41,6 @@ import {WizardStep} from '../util/wizard-step.interface';
 @Component({
   selector: 'aw-wizard-completion-step',
   templateUrl: 'wizard-completion-step.component.html',
-  styleUrls: ['wizard-completion-step.component.css'],
-  encapsulation: ViewEncapsulation.None,
```

Comment:
How are the styles applied if they are not specified via `styleUrls`?

Reply:
Ok I think I can answer this question myself by looking at your PR in the demo repository^^

Reply:
Yes :) Angular no longer knows anything about styles. They are built separately, and have to be included separately by the library user. On the other hand, the user will be able to apply customizations (which is also demonstrated in the demo repo).